themes/icons/gen-ai.svg

```diff
@@ -0,0 +1,4 @@
+<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+<path d="M7.98139 1.04717C7.98805 1.03029 8.01195 1.03029 8.01861 1.04717L9.9799 6.0201L14.9528 7.98139C14.9697 7.98805 14.9697 8.01195 14.9528 8.01861L9.9799 9.9799L8.01861 14.9528C8.01195 14.9697 7.98805 14.9697 7.98139 14.9528L6.0201 9.9799L1.04717 8.01861C1.03029 8.01195 1.03029 7.98805 1.04717 7.98139L6.0201 6.0201L7.98139 1.04717Z" />
+<path d="M2.5 2.65L2.54455 2.55455L2.64 2.51L2.54455 2.46545L2.5 2.37L2.45545 2.46545L2.36 2.51L2.45545 2.55455L2.5 2.65Z" class="filled"/>
```

Comment:
Just a small sanity check, is this little dot expected?

<img width="205" alt="image" src="https://github.com/cloudscape-design/components/assets/812240/80c6484d-3254-4504-9e2b-56cbb8568ff5">

Reply:
renders fine for me in the dev-page preview.
<img width="63" alt="Screenshot 2023-12-21 at 4 03 37 PM" src="https://github.com/cloudscape-design/components/assets/15003460/d07b2991-2ec9-425f-8e80-3af4d5a8696f">